mlflow/protos/service.proto

```diff
@@ -835,6 +838,9 @@ message CreateRun {
   // MLflow release. Use 'mlflow.user' tag instead.
   optional string user_id = 2;
 
+  // Name of the run.
+  optional string run_name = 3;
```

Comment:
For safety, can we make this `optional string run_name = 10`? I don't know if we accidentally deleted items from our proto instead of using the `reserve` keyword (oof), but I suspect that `3` may have previously been used for something. Can you check git history to make sure that 10 is safe?

Reply:
Just checked blame. 3 was indeed used...for run name 😂 
I think we should be safe to use 3 given that any uses of that byte will be for name. Ditto for CreateRun.
For UpdateRun, neither 5 nor name were ever used, so we're fine there.